packages/core/src/Chip/ButtonChip/ButtonChip.tsx

```diff
@@ -20,7 +20,7 @@ const ChipButton = styled(Button)`
   &:focus {
     box-shadow: none;
     outline-offset: 2px;
-    outline: 3px solid ${getPaletteColor('base')};
+    outline: ${(props) => `3px solid ${getPaletteColor('primary.base')(props)}`};
```

Comment:
It looks like the component has a default color prop, should we be using that instead of hardcoding primary?

Reply:
Oh woah that a quick comment. Yeah. I't should just be `getPaletteColor('base')`